modes/modes.py

```diff
@@ -116,27 +116,27 @@ class PoliceMode2(BaseMode):
     def calc_next_step(self):
         if self.step % 4 == 0:
             if self.mid_left:
-                for i in range(self.led_count / 2 - self.mid_width / 2, self.led_count / 2):
-                    self.colors[i] = 0, 0, 0
-                for i in range(self.led_count / 2, self.led_count / 2 + self.mid_width / 2):
-                    self.colors[i] = 0, 0, 254
+                for i in range(int(self.led_count / 2 - self.mid_width / 2), int(self.led_count / 2)):
```

Comment:
there is also the `//` integer-result division operator in py27 and py3.

Reply:
that would be cleaner